senders/types.go

```diff
@@ -28,7 +28,7 @@ type MetricSender interface {
 
 	// Sends a delta counter (counter aggregated at the Wavefront service) to Wavefront.
 	// the timestamp for a delta counter is assigned at the server side.
-	SendDeltaCounter(name string, value float64, source string, tags map[string]string) error
+	SendDeltaCounter(name string, value float64, ts int64, source string, tags map[string]string) error
```

Comment:
Changing this signature will break backwards compatibility for existing users. Is there a need for a timestamp on delta counters?

Reply:
Addd ts for parity with Java and Python SDK (used overloading to give customers the option to add timestamp). Since Go has no overloading, will remove ts parameter.